Boa.Constrictor/Screenplay/Conditions/Strings/IsNullOrWhitespaceCondition.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace Boa.Constrictor.Screenplay
+{
+    public class IsNullOrWhitespaceCondition : ICondition<string>
+    {
+        #region Constructors
+
+        /// <summary>
+        /// Private constructor.
+        /// (Use the public builder method instead.)
+        /// </summary>
+        private IsNullOrWhitespaceCondition()
+        {
+
+        }
+
+        #endregion
+
+        #region Builder Methods
+
+        /// <summary>
+        /// Public builder.
+        /// </summary>
+        /// <returns></returns>
+        public static IsNullOrWhitespaceCondition Value() => new IsNullOrWhitespaceCondition();
+
+        #endregion
+
+        #region Methods
+
+        /// <summary>
+        /// Checks if the actual value equals null or whitespace
+        /// </summary>
+        /// <param name="actual"></param>
+        /// <returns></returns>
+        public bool Evaluate(string actual)
+        {
+            return string.IsNullOrWhiteSpace(actual);
```

Comment:
For consistent coding style, please use the `=>` operator for defining this method.

Reply:
Done